src/servers/src/mysql/server.rs

```diff
@@ -195,12 +199,14 @@ impl MysqlServer {
         spawn_ref: Arc<MysqlSpawnRef>,
         spawn_config: Arc<MysqlSpawnConfig>,
         process_id: u32,
+        prepared_stmt_cache_capacity: usize,
```

Comment:
I think it should be moved into `MysqlSpawnConfig` just like other mysql configurations.

https://greptimedb.rs/servers/mysql/server/struct.MysqlSpawnConfig.html